### ISSUE TITLE
Remove the Exit button

### ISSUE THREAD

**Issue by dainnilsson** [Yubico/yubikey-personalization-gui#14]
The top bar functions as tabs, each element can be selected to show its page. The only exception is Exit, which is the only entry to perform an action (the application exists). This breaks the tab metaphor and should either be removed completely, or separated from the rest of the elements visually (by making it a button, or adding a separator, etc).

Since most window decorators feature a quite prominent exit button (an X in the corner of the window), I feel that our Exit button can safely be removed without hindering our users from closing the app. Less is more.

**Comment by kevinSuttle** [comment]
Good call. Accidentally hit it a few times the first time I ran the software.
